docs/content/stable/architecture/transactions/_index.md

```diff
@@ -77,6 +77,18 @@ YugabyteDB's distributed transaction architecture is based on principles of atom
     </a>
   </div>
 
+  <div class="col-12 col-md-6 col-lg-12 col-xl-6">
+    <a class="section-link icon-offset" href="read-restart-error/">
```

Comment:
This page is not there in stable. Its only in preview https://deploy-preview-22026--infallible-bardeen-164bc9.netlify.app/preview/architecture/transactions/read-restart-error/